src/restaurants/details/DetailPage.tsx

```diff
@@ -18,57 +18,47 @@ const DetailPage = () => {
 	const { setIsError } = useContext(HeaderContext);
 	let { name } = useParams<{ name: string }>();
 
-	const {
-		data: restaurantData,
-		isLoading: isRestaurantLoading,
-		error: restaurantError,
-		isError: restaurantIsError
-	} = getMenuPerRestaurantQuery(name!);
 	const {
 		data: menuData,
 		isLoading: isMenuLoading,
 		error: menuError,
 		isError: menuIsError
```

Comment:
same case here, no need to add 'menu' to everything, we could just have isLoading, error, isError, etc since it is the only query we're using within this component